afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java

```diff
@@ -116,20 +120,28 @@ private ModifiedNetwork loadNetworkFromProjectCase(ProjectCase projectCase) {
         }
     }
 
-    private <T extends ProjectFile & ProjectCase> ModifiedNetwork loadNetwork(T projectCase) {
+    private ModifiedNetwork loadNetwork(T projectCase) {
```

Comment:
This method should be static or remove the projectCase parameter to use the projectCase member.

Reply:
Done